src/ape/api/projects.py

```diff
@@ -131,8 +131,8 @@ def _create_source_dict(
         source_dict: Dict[str, Source] = {}
         for source_path in contract_filepaths:
             key = str(get_relative_path(source_path, base_path))
-            source_dict[key] = Source(  # type: ignore
```

Comment:
Some of these ignores were here from before we started using pydantic, so they might not be necessary anymore